lib/cs_guide/accounts/user.ex

```diff
@@ -4,7 +4,9 @@ defmodule CsGuide.Accounts.User do
   import Ecto.Changeset
 
   schema "users" do
-    field(:email, :string)
+    field(:email, Fields.EmailEncrypted)
```

Comment:
In the interest of _shipping_ the CS app, I support this. 👍 
We need to _refactor_ any user-related code into an Umbrella App called "Auth" with a _separate_ Repo which we then "hand off" any email related requests to. 
More on that later. focus on shipping CS. 👨‍💻